src/api/public/apidocs-new/paths/source_project_name_attribute.yaml

```diff
@@ -0,0 +1,54 @@
+get:
+  summary: Get all the project's attributes
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - name: meta
+      in: query
+      schema:
+        type: string
```

Comment:
I miss for these parameters (meta, rev, view, with_default, and with_project):
- to fill out the `description:` field
- a definition of values they could have. For example, if they are booleans
- an example value

Reply:
Done.
I documented only the query parameters that can be used at this project level. Some others don't make sense or make the code crash, they are commented on but not documented.